client/src/components/LoginTest/login.js

```diff
@@ -1,6 +1,6 @@
 // import React from "react";
 // import './login.css'
-import React, { Component, Fragment } from "react";
+import React, { Fragment } from "react";
```

Comment:
not sure who originally added 'Component' so idk if i can approve the removal. @water96763 @Spacejnk @czjordan323 ?